google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -718,6 +729,129 @@ public void testJsonStreamWriterWithDefaultStream()
     }
   }
 
+  @Test
+  public void testJsonDefaultStreamOnTableWithDefaultValue_SchemaNotGiven()
+      throws IOException, InterruptedException, ExecutionException,
+          Descriptors.DescriptorValidationException {
+    String tableName = "defaultStreamDefaultValue";
+    String defaultTableId =
+        String.format(
+            "projects/%s/datasets/%s/tables/%s",
+            ServiceOptions.getDefaultProjectId(), DATASET, tableName);
+    tableInfo =
+        TableInfo.newBuilder(
+                TableId.of(DATASET, tableName),
+                StandardTableDefinition.of(
+                    Schema.of(
+                        com.google.cloud.bigquery.Field.newBuilder(
+                                "foo_with_default", LegacySQLTypeName.STRING)
+                            .setDefaultValueExpression("'default_value_for_test'")
+                            .setMode(Field.Mode.NULLABLE)
+                            .build(),
+                        com.google.cloud.bigquery.Field.newBuilder(
+                                "bar_without_default", LegacySQLTypeName.STRING)
+                            .setMode(Mode.NULLABLE)
+                            .build(),
+                        com.google.cloud.bigquery.Field.newBuilder(
+                                "date_with_default_to_current", LegacySQLTypeName.DATETIME)
+                            .setDefaultValueExpression("CURRENT_DATE()")
```

Comment:
CURRENT_TIMESTAMP? That is the most common usage.

Reply:
Done